### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gitgoap**
Fixed small grammatical mistake.

This change makes the sentence look a little better (may be 0.1% impact)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ To ensure consistency throughout the source code, please keep these rules in min
 
 ## Need help? Questions and suggestions
 
-Questions, suggestions, and thoughts are most welcome. Feel free to open a [GitHub Issue](https://github.com/novuhq/novu/issues/new/choose). We can also be reached in our [Discord Server](https://discord.novu.co).
+Questions, suggestions, and thoughts are most welcome. Feel free to open a [GitHub Issue](https://github.com/novuhq/novu/issues/new/choose). We can also be reached on our [Discord Server](https://discord.novu.co).
```
Comment (scopsy) [inline CONTRIBUTING.md:56]: @Cliftonz is there a grammatical issue here? Always confusing those 😅

Comment (gitgoap) [reply CONTRIBUTING.md:56]: May be it's not a grammatical error but it surely enhance this line.

Comment (Cliftonz) [reply CONTRIBUTING.md:56]: @scopsy This is correct, "on" would be the better phrasing for what we are tring to convey.

**Review by Cliftonz — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (onlymealissa86-code) [reply CONTRIBUTING.md:56]: Yeah always confused...little bugger

**Pull request merged**